apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
I admit I don't really understand the idea about Services/Managers layer neither :/  Seems these 2 layer represent the wrapper for Bridge library client/service like @azasypkin  said:
view - (service layer) - (manager layer) - workers